coordinator/core/src/main/kotlin/net/consensys/zkevm/ethereum/coordination/aggregation/GlobalAggregationCalculator.kt

```diff
@@ -24,6 +24,9 @@ class GlobalAggregationCalculator(
     b1.startBlockNumber.compareTo(b2.startBlockNumber)
   }
 
+  private val aggregationSizeInBlocks = AtomicInteger(0)
```

Comment:
I think gauge doesn't fit well what we're trying to capture here. 

I think histogram is more suitable, because:
* We won't lose any data in case multiple aggregations are processed in a short period of time (less than scrape interval)
* It will give us much more information on the distribution of block / blob / block sizes per aggregation